components/form/captcha/src/index.js

```diff
@@ -25,6 +26,9 @@ const FormCaptcha = ({
   const [showCaptcha, setShowCaptcha] = useState(false)
   const [captchaId, setCaptchaId] = useState(null)
 
+  // If we receive a unique identifier, we concatenate it with the default.
+  const captchaContainerId = [CAPTCHA_ID, containerId].filter(Boolean).join(' ')
```

Comment:
🍆 why don't you just use one (the default one) or the other instead of joining them together?

Reply:
Well, for nothing special, but as it refers to the class of the component, I wanted to keep it. 😛

Reply:
I wouldn't have used a blank space to join them... it's an id! ;)